components/files/file_upload/file_upload.py

```diff
@@ -27,6 +29,11 @@ def path_up_to_last(a_last, a_inclusive=True, a_path=os.path.dirname(os.path.rea
 # This is the mechanism by which file versioning is achieved.
 #
 # Because the FileUploadbean is the start of the upload_uuid chain, it is generated on creation rather than passsed in as a client parameter.
+#
+# Sitelink3D v2 will reject file uploads larger than 10485760 bytes. In these cases, the repsponse will be of the form 
+# 400:{"success":false,"error":"Upload size (41455634) exceeds maximum size (10485760) for a single part upload","preventRetry":false}
+# For this reason, multi-part uploads should be used for files in excess of this size. This is demonstrated in the below example.
+#

```

Comment:
FileUploadBean.to_json seems is not used anymore, may be remove FileUploadBean, or refactor upload_file_multipart to use FileUploadBean instead?

Reply:
Refactored as the bean class is used more extensively throughout the examples and should also be leveraged here.